pyproject.toml

```diff
@@ -0,0 +1,34 @@
+[tool.poetry]
+name = "tom-demo"
+version = "1.0.7"
```

Comment:
Not a show-stopper, but this might be an opportunity to use `poetry-dynamic-versioning`.